.github/workflows/test_datasets.yml

```diff
@@ -1,12 +1,12 @@
-name: datasets validation
+name: datasets CRON
```

Comment:
Note: it's not VRON only. If a PR modifies `sktime/base` or `sktime/datasets`, this workflow is triggered by the code quality workflow.

Reply:
updated